layers/shaders/memory_decompression/spirv/GInflate16_HAVE_INT16_HAVE_INT64_vk.h

```diff
@@ -0,0 +1,925 @@
+// 1112.0.0
```

Comment:
In addition to copyrights, you should add comments to these files noting that they are generated from your .glsl files.

Reply:
These files are output of glslangValidator, so it would involve some additional scripting to add copyright/autogen info.
I guess a better solution would be to make it part of cmake. Could that be done as a followup change?